charts/grafeas-elasticsearch/templates/deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "grafeas.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "grafeas.name" . }}
+    helm.sh/chart: {{ include "grafeas.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "grafeas.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "grafeas.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      initContainers:
+        - name: wait-for-elasticsearch
+          image: curlimages/curl:7.73.0
+          imagePullPolicy: IfNotPresent
+          command:
+            - "/bin/sh"
+            - "-c"
+            - |
+              until curl --output /dev/null --silent --fail --max-time 2 {{ .Values.grafeas.elasticsearch.url }}/_cluster/health?wait_for_status=yellow; do sleep 1; done;
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/grafeas-server"]
+          args: ["--config=/etc/config/config.yaml"]
+          ports:
+            - name: grafeas
+              containerPort: {{ .Values.container.port }}
+              protocol: TCP
+          volumeMounts:
+          - name: config-volume
+            mountPath: /etc/config
+{{- if eq .Values.certificates.enabled true }}
+          - name: certificates
+            mountPath: /certificates
+{{- end }}
+{{- if .Values.persistence.enabled }}
+{{- if eq .Values.persistence.storageType "embedded" }}
```

Comment:
I don't think this is needed for our chart - we can probably assume that all users will be using the `elasticsearch` storage type.

Reply:
Removed.